src/components/Dropdown/Dropdown.tsx

```diff
@@ -285,44 +344,66 @@ export default class Dropdown extends AutoControlledComponent<
           predefinedProps,
           highlightedIndex,
           selectItemAtIndex,
-          getRootProps,
+          accessibilityComboboxProps,
           getInputProps,
         ),
     })
   }
 
   private renderToggleButton(
     getToggleButtonProps: (options?: GetToggleButtonPropsOptions) => any,
-    styles: ComponentSlotStylesInput,
+    classes: ComponentSlotClasses,
     isOpen: boolean,
   ) {
+    const { onClick } = getToggleButtonProps()
     return (
-      <Icon
-        name={`chevron ${isOpen ? 'up' : 'down'}`}
-        as="button"
-        tabIndex="-1"
-        styles={styles.toggleButton}
-        {...getToggleButtonProps()}
-      />
+      <span className={classes.toggleButton} onClick={onClick}>
+        {isOpen ? String.fromCharCode(9650) : String.fromCharCode(9660)}
+      </span>
     )
   }
 
   private renderItemsList(
     styles: ComponentSlotStylesInput,
     variables: ComponentVariablesInput,
-    getMenuProps: (options?: GetMenuPropsOptions, otherOptions?: GetPropsCommonOptions) => any,
-    getItemProps: (options: GetItemPropsOptions<ShorthandValue>) => any,
     isOpen: boolean,
     highlightedIndex: number,
+    toggleMenu: () => void,
+    selectItemAtIndex: (index: number) => void,
+    getMenuProps: (options?: GetMenuPropsOptions, otherOptions?: GetPropsCommonOptions) => any,
+    getItemProps: (options: GetItemPropsOptions<ShorthandValue>) => any,
+    getInputProps: (options?: GetInputPropsOptions) => any,
   ) {
-    const accessibilityMenuProps = getMenuProps({ refKey: 'innerRef' })
+    const accessibilityMenuProps = {
+      ...getMenuProps({ refKey: 'innerRef' }, { suppressRefError: true }),
+    }
```

Comment:
```suggestion
 const accessibilityMenuProps = getMenuProps({ refKey: 'innerRef' }, { suppressRefError: true })
```

Destruction doesn't make any sense there